src/pages/messages/TasksPage.tsx

```diff
@@ -180,8 +176,8 @@ const TasksPage = () => {
 
   const notificationsQuery = useQuery({
     enabled: isOnTicketsPage && !institutionUserQuery.isLoading,
-    queryKey: ['notifications', notificationsParams],
-    queryFn: () => fetchCustomerTickets(notificationsParams),
+    queryKey: ['notifications', taskNotificationsParams],
```

Comment:
Denne må vel også ha 'task' i queryKey om du tar til følge kommentarene over.
```suggestion
    queryKey: ['notifications', 'task', taskNotificationsParams],
```